tests/libvmi/vmi.go

```diff
@@ -71,6 +71,14 @@ func WithResourceMemory(value string) Option {
 	}
 }
 
+// WithNamespace uses 'namespace' arg as the vmi
+// namespace.
```

Comment:
can we combine those 2 comment lines to one please?

Reply:
do we use this function ?
 WaitUntilVMIReady doesnt need a namespace anymore

Reply:
You right, forgot to remove this after moving back to cirros, thanks.